leakcanary-android-sample-tv/build.gradle

```diff
@@ -0,0 +1,60 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
```

Comment:
Could this be a flavor of the example app instead?

Reply:
I don't see why not. I can modify the regular sample app to work both on tv and mobile. Not a flavor, it will be same APK, just with extra code to run for TV.

Reply:
Works for me!

Reply:
Added a button that leaks an activity (by recreating it) - it's very easy to click it on TV; works for mobile as well

Reply:
>  I can modify the regular sample app

why is there still a leakcanary-android-sample-tv ?

Reply:
Oops, I need to do some clean up